sdk/iOS/test/MSUserAgentBuilderTests.m

```diff
@@ -33,7 +33,7 @@ -(void) testURLForTable
 {
     NSString *userAgent = [MSUserAgentBuilder userAgent];
     
-    XCTAssertTrue([userAgent isEqualToString:@"ZUMO/2.1 (lang=objective-c; os=--; os_version=--; arch=iOSSimulator; version=2.1.0)"],
+    XCTAssertTrue([userAgent isEqualToString:@"ZUMO/3.0 (lang=objective-c; os=--; os_version=--; arch=iOSSimulator; version=3.0.0)"],
```

Comment:
did we say this shouldn't go to dev as well?  Thinking on this changeset, it needs to go into dev as well.